website/docs/api/plugin-methods/lifecycle-apis.md

```diff
@@ -324,6 +324,43 @@ module.exports = function (context, options) {
 };
 ```
 
+## `postStart(props)` {#postStart}
+
+Called when a (development) server finishes starting.
```

Comment:
"finishes starting" is a bit unclear to me. Start can mean a lot of things like server has opened its port or bundler has completed its first build? 🤷‍♂️